### PR TITLE
Add TagTci command.

### PR DESCRIPTION
**Pull request by zhalvorsen**

### CODE REVIEW

**Review by ia0 — APPROVED** [review]

File: dpe/src/dpe_instance.rs [hunk 1]
```diff
@@ -144,6 +147,46 @@ impl<C: Crypto> DpeInstance<'_, C> {
...
+        }
+
+        context.has_tag = true;
+        context.tag = cmd.tag;
+
+        // Because handles are one-time use, let's rotate the handle, if it isn't the default.
+        context.handle = match context.context_type {
+            ContextType::_Normal | ContextType::Simulation => {
+                let mut handle = [0; HANDLE_SIZE];
+                C::rand_bytes(&mut handle)?;
```
[the remark this answers is not in the document]
Comment (jhand2) [reply dpe/src/dpe_instance.rs:179]: We'll have to do this for default contexts anyway. Agreed this is probably a good idea.

I think the performance hit is pretty minimal and the leaked information isn't really a big deal.

[the remark this answers is not in the document]
Comment (zhalvorsen) [reply dpe/src/dpe_instance.rs:179]: Would it be better to loop forever or have a max attempts? Either way if it has to try more than 2 or 3 times there is probably something major wrong.

[the remark this answers is not in the document]
Comment (jhand2) [reply dpe/src/dpe_instance.rs:179]: Agreed, failing after a couple tries makes sense.

[the remark this answers is not in the document]
Comment (zhalvorsen) [reply dpe/src/dpe_instance.rs]: I added a helper function. I started running into the problem with mixing mutable and non-mutable references so I had to refactor a little. Let me know what you think.

**Review by jhand2 — APPROVED** [review]

**Pull request merged**